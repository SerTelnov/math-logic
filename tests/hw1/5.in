A|-B
A
A->B
B
A