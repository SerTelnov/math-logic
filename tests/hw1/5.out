(1) A (Предп. 1)
(2) (A->B) (Не доказано)
(3) B (M.P. 2, 1)
(4) A (Предп. 1)